scripts/scripts.js

```diff
@@ -167,9 +167,9 @@ async function loadEager(doc) {
   decorateTemplateAndTheme();
   const main = doc.querySelector('main');
   if (main) {
-    await decorateTemplates(main);
     decorateMain(main);
     document.body.classList.add('appear');
+    await decorateTemplates(main);
```

Comment:
I moved template decoration after decorateMain so that all the blocks would already be decorated. This made it easier to manipulate the DOM for the template. Please let me know if there are concerns with doing this, and I can revisit it.